process.py

```diff
@@ -0,0 +1,11 @@
+import sys
+
+with open(sys.argv[1], 'r') as f: 
+    contents = f.readlines()
+    data = []
+    for line in contents:
+        data.append(line.split())
+
+with open('odgi_output.txt', 'w') as f2:
+    for i in range(1, len(data)):
+        f2.write(f'{data[i][0].strip()} {data[i][1].strip()}\n')
```

Comment:
I'd also avoid hard-coding the output filename here too. Just using a normal `print` will make this print to the standard output stream, making things more flexible.